content/shmem_team_destroy.tex

```diff
@@ -1,27 +1,27 @@
 \apisummary{
-    shmem\_team\_destroy function destroys existing team.
+    Destroys existing team.
 }
 
 \begin{apidefinition}
 
 \begin{Csynopsis}
-int @\FuncDecl{shmemx\_team\_destroy}@(shmem_team_t newteam);
+int @\FuncDecl{shmem\_team\_destroy}@(shmem_team_t team);
 \end{Csynopsis}
 
 \begin{apiarguments}
-\apiargument{IN}{newteam}{A valid SHMEM team handle.}
+\apiargument{IN}{team}{A valid SHMEM team handle.}
 \end{apiarguments}
 
 \apidescription{
-The shmemx\_team\_destroy function destroys an existing team. This is a
+The shmem\_team\_destroy function destroys an existing team. This is a
 collective call, in which every member of the team being destroyed needs
 to participate. This will free all internal memory structures associated
 with the team and invalidate the team handle. Upon return, the team
-handle is set to SHMEM\_TEAM\_NULL, after which it can no longer be
-used for team collective calls.
+handle is set to \CONST{SHMEM\_TEAM\_NULL}, after which it can no longer be
+used for team API calls.
 
-It is considered erroneous to free SHMEM\_TEAM\_WORLD or
-SHMEM\_TEAM\_NODE. Error checking will be done to ensure a valid
+It is considered erroneous to free \CONST{SHMEM\_TEAM\_WORLD} or
+\CONST{SHMEM\_TEAM\_NODE}. Error checking will be done to ensure a valid
```

Comment:
IIRC, indexing was new in 1.4, and I'm not sure whether `\CONST{...}` or `\LibConstRef{...}` is more appropriate here. Want to chime in here, @BryantLam?